endpoints.json

```diff
@@ -55,5 +55,31 @@
       "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
       "comment_count": 2
     }
+  },
+  "GET /api/articles/:article_id/comments":{
+    "description":"Takes an existing article Id number and returns its comments if existing, if not returns an empty array",
+    "queries":[  "author",
```

Comment:
not sure if we will be querying for all of these with regards to the comments, it's best to leave this empty till we've added the ability to query by anything o that we only add the relevant query options here :)